src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -959,6 +959,17 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         AdjustFontSize(fontDelta);
     }
 
+    // Method Description:
+    // - Reset the font size of the terminal to its default size.
+    // Arguments:
+    // - none
+    void TermControl::ResetFontSize()
+    {
+        const auto currentSize = _desiredFont.GetEngineSize().Y;
+        const auto defaultSize = _settings.FontSize();
+        AdjustFontSize(defaultSize - currentSize);

```

Comment:
_nit_: I think I'd prefer if we changed this code up a bit to have a `_SetFontSize` method that `ResetFontSize` and `AdjustFontSize` both call, because using `AdjustFontSize` to set the font size like this _feels_ wonky, but it works so I'm not gonna block over this.

Reply:
Thanks for the comment, I also thought about this, but I didn't want to make big changes which might affect things around.
anyway, I`ll make the needed changes and update the PR.